### PR TITLE
Adding support for node 12 (LTS)

### PR DESCRIPTION
**Pull request by ahmetcetin**
As Node 12 is now LTS, adding it to the engine

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "3.4.0",
   "description": "Current context for LoopBack applications, based on cls-hooked",
   "engines": {
-    "node": "^8.2.1 || ^10.14"
+    "node": "^8.2.1 || ^10.14" || "^12.13"
```
Comment (bajtos) [inline package.json:6]:
```suggestion
    "node": "^8.2.1 || ^10.14 || ^12.13"
```

**Review by bajtos — CHANGES_REQUESTED** [review]
@ahmetcetin thank you for the pull request ❤️ 

First of all, please fix the JSON syntax, quoting from the failed CI build:

```
npm ERR! JSON.parse Failed to parse json
npm ERR! JSON.parse Unexpected token | in JSON at position 183 while parsing near '... "^8.2.1 || ^10.14" || "^12.13"
npm ERR! JSON.parse   },
npm ERR! JSON.parse   "...'
npm ERR! JSON.parse Failed to parse package.json data.
npm ERR! JSON.parse package.json must be actual JSON, not just JavaScript.
```

Secondly, add Node.js 12 to https://github.com/strongloop/loopback-context/blob/master/.travis.yml. If we want to claim Node.js 12 as supported, then we need to verify that claim as part of our CI.

Last but not least: this package is in LTS mode, which means it does not accept new features. Adding support for a new Node.js version is technically a new feature. Having said that, if Node.js 12 works out of the box (or with minimal changes), then I am fine to include it in the supported versions.

**Comment by ahmetcetin** [comment]
Hi @bajtos, 
Sorry for being unresponsive. It looks I lost the notification email from github. I totally forgot about this PR since then, and opened another one #54. So I close this one in favour of the other one. Indeed this package works fine with any node version above 8.

**Pull request closed without merging**